Changelog.md

```diff
@@ -7,6 +7,7 @@ Compiler Features:
 * Commandline Interface: Add `--no-cbor-metadata` that skips CBOR metadata from getting appended at the end of the bytecode.
 * Standard JSON: Add a boolean field `settings.metadata.appendCBOR` that skips CBOR metadata from getting appended at the end of the bytecode.
 * Yul Optimizer: Allow replacing the previously hard-coded cleanup sequence by specifying custom steps after a colon delimiter (``:``) in the sequence string.
+* Language Server: Fixes URL decoding of incoming file names.
```

Comment:
I was initially confused as to why file names would need URL-decoding. I think it would be better to say that we're decoding the `file://` URLs.
```suggestion
* Language Server: Properly URL-decode the `file://` URIs coming from the client and encode the URIs coming out of by the server.
```
I'd also say it's a bugfix rather than a feature :)